rules/handle-noise-email/rule.md

```diff
@@ -0,0 +1,26 @@
+---
+type: rule
+title: Do you know how to handle noise email
+guid: e0d0429d-79a7-4005-a683-20f4a6671d08
+uri: handle-noise-email
+created: 2023-10-25T04:05:17Z
+authors: 
+  - title: Ben Neoh
+    url: https://www.ssw.com.au/people/ben-neoh/
+related:
```

Comment:
add this rule to related
https://www.ssw.com.au/rules/clean-your-inbox-per-topics/